reference/api/documents.md

```diff
@@ -128,9 +128,12 @@ This endpoint accepts the following content types:
 | Query Parameter  | Default Value | Description                                                                   |
 | :--------------- | :------------ | :---------------------------------------------------------------------------- |
 | **`primaryKey`** | `null`        | [Primary key](/learn/core_concepts/primary_key.md#primary-key-2) of the index |
+| **`csvDelimiter`** | `","`        | Configure the character separating CSV fields. Must be a string containing [one ASCII character](https://www.rfc-editor.org/rfc/rfc20). |
 
 If you want to [set the primary key of your index on document addition](/learn/core_concepts/primary_key.md#setting-the-primary-key-on-document-addition), it can only be done **the first time you add documents** to the index. After this, the `primaryKey` parameter will be ignored if given.
 
+Configuring `csvDelimiter` and sending data in a format other than CSV will cause Meilisearch to throw an error.
```

Comment:
Don’t know if it’s worth documenting it, but technically we will not even read the data sent by the user. We will only read the content type.

```suggestion
Configuring `csvDelimiter` and sending data with a content type other than CSV will cause Meilisearch to throw an error.
```